app/bettermail.go

```diff
@@ -274,6 +285,9 @@ func handleCommitCommentPayload(payload CommitCommentPayload, c context.Context)
 		SenderUserName: senderUserName,
 		Subject: subject,
 		HTMLBody: mailHtml.String(),
+		Headers: map [string]string {
+			"In-Reply-To": messageId,
```

Comment:
Yeah. It seems to work fine in gmail even if this header is empty, but who knows what mail client it might break.